.github/workflows/maven.yml

```
@@ -44,7 +44,7 @@ jobs:
             maven-${{ matrix.os }}-
 
       - name: Set up JDK
-        uses: actions/setup-java@v1
+        uses: actions/setup-java@v2
         with:
           java-version: ${{ matrix.java }}
```

Needs to specify the distribution.
```suggestion
          distribution: 'adopt'
          java-version: ${{ matrix.java }}
```